tests/features/test_features.py

```diff
@@ -0,0 +1,99 @@
+"""Miscelaneous tests of features."""
+from pathlib import Path
+from itertools import chain
+
+import numpy as np
+import pytest
+from numpy import testing as npt
+
+import neurom as nm
+from neurom import features
+
+
+@pytest.fixture
+def DATA_PATH():
+    return Path(__file__).parent.parent / "data"
+
+
+@pytest.fixture
+def SWC_PATH(DATA_PATH):
+    return DATA_PATH / "swc"
+
+
+@pytest.fixture
+def MORPHOLOGY(SWC_PATH):
+    return nm.load_morphology(SWC_PATH / "test_morph.swc")
+
+
+@pytest.fixture
+def NEURITE(MORPHOLOGY):
+    return MORPHOLOGY.neurites[0]
+
+
+@pytest.fixture
+def SECTION(NEURITE):
+    return NEURITE.sections[0]
+
+
+@pytest.fixture
+def NRN_FILES(DATA_PATH):
+    return [
+        DATA_PATH / "h5/v1" / f for f in ("Neuron.h5", "Neuron_2_branch.h5", "bio_neuron-001.h5")
+    ]
+
+
+@pytest.fixture
+def POP(NRN_FILES):
+    return nm.load_morphologies(NRN_FILES)
+
+
+def _check_nested_type(data):
+    """Check that the given data contains only built-in types.
+
+    The data should either be an int or float, or a list or tuple of ints or floats.
+    """
+    if isinstance(data, (list, tuple)):
+        for i in data:
+            _check_nested_type(i)
+    else:
+        assert isinstance(data, (int, float))
+
+
+class TestFeatureTypes:
```

Comment:
Is the class necessary here? Could you list the tests without the class definition?

Reply:
It's not necessary, it's just to organize the tests. I can remove it if you prefer.

Reply:
Yeah, I think that the tests are quite descriptive by themselves.

Reply:
Done